packages/fields/src/actions/index.ts

```diff
@@ -1,13 +1,10 @@
 export { default as viewPost } from './view-post';
 export { default as reorderPage } from './reorder-page';
-export { default as reorderPageNative } from './reorder-page.native';
```

Comment:
@dcalhoun sorry to ping you so many times lately :)

Is it ok to remove the native actions here? They export `undefined` ([reorder](https://github.com/WordPress/gutenberg/pull/67580/files#diff-92034ee995c7df98ec7822103430854fdbfba95884a33c2207b89ccf8d957b2d)), [export pattern](https://github.com/WordPress/gutenberg/pull/67580/files#diff-adb87917ac23eb2f91ab752cbf792b63ef7afff13e909812384b193c57727529), [duplicate post](https://github.com/WordPress/gutenberg/pull/67580/files#diff-bee647c4d0b8f8af18507649b5372b0b6427660454a84d055ae80b0dc8be4579), so we wonder why are they necessary?

Reply:
Mostly to make unit tests pass. If nothing breaks, it's ok to remove IMO

Reply:
I'll concede to @youknowriad's knowledge on this one. It appears the addition occurred in https://github.com/WordPress/gutenberg/pull/64199/commits/94d1d2b7f1095fc5692f0f073160badfcbd0130f to repair unit test failures. I agree it is safe to remove if unit tests continue passing.

Reply:
Thanks for the confirmation to both of you 🙇